wasmer/wasmer_link_static.go

```diff
@@ -0,0 +1,6 @@
+// +build static_build
```

Comment:
See comment on wasmer_link_dynamic.go. underscores should be removed from file names unless they are intended to have meaning as tags.

Reply:
See my remark above please.